dev/src/write-batch.ts

```diff
@@ -509,6 +543,51 @@ export class WriteBatch {
     return this.commit_();
   }
 
+  /**
+   * Commits all pending operations to the database and verifies all
+   * preconditions. Fails the entire write if any precondition is not met.
+   *
+   * The writes in the batch are not applied atomically and can be applied out
+   * of order.
+   *
+   * @private
+   */
+  async batchCommit(commitOptions?: {
```

Comment:
What do you think of the name "bulkCommit"? I would like to reduce confusion, and deciding between "commit" and "batchCommit" is going to be tough.

Reply:
I like that more -- the hard part was figuring out where to draw the line since we refer to it externally as bulk writes, but the backend API refers to it as batch writes.